boards/lilygo/ttgo_t8c3/doc/index.rst

```diff
@@ -0,0 +1,240 @@
+.. _ttgo_t8c3:
+
+Lilygo ttgo t8-c3
```

Comment:
Should you use T8-C3 in upper case instead? At least this is the way it is written in documentation (https://github.com/Xinyuan-LilyGO/T8-C3). Anyway, not mandatory at all.

Reply:
+1, upper case would be preferred IMO